### PR TITLE
Updated tests

### PR DESCRIPTION
**Pull request by snapperVibes**
The old tests didn't work as @__made new tweets and @kennethreitz made their account private.

### CODE REVIEW

File: test.py [hunk 1]
```diff
@@ -14,11 +14,11 @@ def test_mother(self):
         user = '__'
         tweets = list(get_tweets(query=user, pages=1))
 
-        self.assertTrue(tweets[0]['text'].__contains__('It is a gift to be alive in the time of Beyoncé'))
+        self.assertTrue(tweets[3]['text'].__contains__('It is a gift to be alive in the time of Beyoncé'))
 
     def test_child(self):
         user = '___'
-        tweets = list(get_tweets(query=user, pages=1))
+        tweets = list        (get_tweets(query=user, pages=1))
```
Comment (bisguzar) [inline test.py]: Why you leave a tab between **list** and **(** ?

Comment (snapperVibes) [reply test.py]: Beginner's error. This was my first pull request and I made a goof while editing.

Comment (bisguzar) [reply test.py]: It's okay, not need new one.

**Comment by bisguzar** [comment]
Nice catch, thanks for your efforts ^^

**Pull request merged**